tox.ini

```diff
@@ -0,0 +1,47 @@
+[tox]
+envlist = py,coverage-report,typing,lint,docs,docs-linkcheck
+isolated_build = True
+
+[testenv]
+description = Run pytest against {envname}.
```

Comment:
I'm a little worried that the testing here is going to conflict with how we want/need to do testing in per-component subdirectories.  Can we leave this out for a docs-only commit?

Reply:
Unless you plan to use `tox` for running tests, then this file should not affect anything else. Pytest won't look at this file if you run it directly.